### ISSUE TITLE
Scalars stopped working in 1.5.0

### ISSUE THREAD

**Issue by Aleksion** [dotansimha/graphql-code-generator#2342]
**Describe the bug**
 
When I run the codegen i'm getting the following error:
```
 ✔ Loading Apollo Project
  ✔ Saving schema to ./src/graphql/server/schema.json
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/GeneratedComponents.tsx
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → Unknown type "DateTime".
    ❯ Generate src/GeneratedPrismicComponents.tsx
      ⠙ Load GraphQL schemas
        Load GraphQL documents
  ✔ Parse configuration
  ❯ Generate outputs    ❯ Generate src/GeneratedComponents.tsx
      ✔ Load GraphQL schemas      ✔ Load GraphQL documents
      ✖ Generate        → Unknown type "DateTime".
    ✔ Generate src/GeneratedPrismicComponents.tsx
 Found 1 error

  ✖ src/GeneratedComponents.tsx
    Error: Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "Date".

    Unknown type "Date".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".

    Unknown type "DateTime".


Something went wrong
```

The Codegen config looks as follows:

```
   plugins:
      - add: "// THIS IS A GENERATED FILE, DO NOT EDIT IT!"
      - add: /* tslint:disable */
      - add: import * as ReactApollo from "react-apollo"
      - typescript
      - typescript-operations
      - typescript-react-apollo
    config:
      noNamespaces: true
      withHooks: true
      withComponent: false
      withHOC: false
      withMutationFn: false
      skipTypename: true
      avoidOptionals: false
      scalars:
        DateTime: string
        Date: string
```

**To Reproduce**
Upgrade lib from 1.3.0 -> 1.5.0

1. My GraphQL schema:

```graphql
# Put your schema here
```

2. My GraphQL operations:

```graphql
# Put your operations here
```

3. My `codegen.yml` config file:

```yml
# Put your YML here
```

**Expected behavior**

**Environment:**

- OS: 
- `@graphql-codegen/...`: 
- NodeJS: 

**Additional context**

**Comment by dotansimha** [comment]
@Aleksion can you please try the latest alpha? `1.5.1-alpha-46a8a9f9.21`
If it's still not working, can you please create a reproduction in a repo or a sandbox?

**Comment by Aleksion** [comment]
That works! :)

**Comment by dotansimha** [comment]
Fixed in 1.6.0.
